src/content/drafts/vue-rules-better-life.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Vue Uncategorized rules to have a better life
+teaser: "Some useful Uncategorized Vue rules that will make your daily life a little easier."
+authors:
+  - BobToninho
+categories:
+  - Case Studies
+tags:
+  - Guest Post
+  - Rules
+---
+
+## Introduction
+
+The [esling-plugin-vue](https://eslint.vuejs.org/) plugin divides its rules in the following categories:
+
+- Base
+- Essential
+- Strongly recommended
+- Recommended
+- Uncategorized
+- Extensions
+
+I spent some time digging into the *uncategorized* category to see what can be useful to help avoid distractions in commits and, of course, to enforce the same style in all parts of a Vue project. In my opinion, all Vue projects should use the Recommended category, which enforces a lot (if not all) of the topics mentioned in the [style guide](https://vuejs.org/style-guide/).
```

Comment:
Some cleanup, and the last sentence doesn’t fit with the paragraph or the rest of the article. 

```suggestion
I spent some time digging into the uncategorized category to see which rules might be useful to help avoid problems in commits and to enforce the same style in all parts of a Vue project. In my opinion, all Vue projects should use the Recommended category, which enforces a lot (if not all) of the topics mentioned in the [style guide](https://vuejs.org/style-guide/).
```